spec/models/grda_warehouse/hud/client_spec.rb

```diff
@@ -153,6 +154,60 @@
         expect(GrdaWarehouse::Hud::Client.full_housing_release_on_file.count).to eq(1)
       end
     end
+
+    context 'when consent validity is two years' do
+      before(:each) do
+        client.instance_variable_set(:@release_duration, 'Two Years')
+        client_signed_yesterday.instance_variable_set(:@release_duration, 'Two Years')
+        client_signed_2_years_ago_short_consent.instance_variable_set(:@release_duration, 'Two Years')
+        client_signed_3_years_ago_short_consent.instance_variable_set(:@release_duration, 'Two Years')
```

Comment:
Should that be 'Three Years'?

Reply:
I don't think so, the release duration is only Two years.  One of the clients has a 3 year expiration date though.